src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,7 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="SynchronizingWithClassView" xml:space="preserve">
     <value>Synchronizing with {0}...</value>
   </data>
+  <data name="FullSolutionAnalysisOff" xml:space="preserve">
+    <value>Full solution analysis disabled due to low memory. go to tools-&gt;options and search "Enable full solution analysis" to turn it back on or restart VS to restore full solution analysis.</value>
```

Comment:
capital G in "go to".
"and search for"

"and search for "Enable full solution analysis" or restart VS to restore full solution analysis."

--

Can i get a screen shot of what this looks like when it is presented?